base64/decode/decode.go

```diff
@@ -0,0 +1,38 @@
+package decode
```

Comment:
Why does it sit in decode package and it's called a service instead of sitting in base64 package being called a decoder.